test/process_test.cc

```diff
@@ -88,10 +126,17 @@ INSTANTIATE_TEST_SUITE_P(IpVersions, ProcessTest,
                          Envoy::TestUtility::ipTestParamsToString);
 
 TEST_P(ProcessTest, TwoProcessInSequence) {
+  FakeStatsSinkFactory factory;
+  Envoy::Registry::InjectFactory<NighthawkStatsSinkFactory> registered(factory);
+  numFlushes = 0;
```

Comment:
If we need to manually reset it to zero at the beginning of a test case, we might be better off doing it in the SetUp method on the test fixture (ProcessTest).

https://github.com/google/googletest/blob/master/googletest/docs/primer.md#test-fixtures-using-the-same-data-configuration-for-multiple-tests-same-data-multiple-tests

Reply:
The original intention is to make it obvious that `numFlushes` will get updated after `runProcess`. Since now we added the new test case for testing stats sink, and not all test cases need to set `numFlushes = 0`, so I think it is better to leave it in the test case.